complex.js

```diff
@@ -1238,6 +1317,8 @@
   Complex['I'] = new Complex(0, 1);
   Complex['PI'] = new Complex(Math.PI, 0);
   Complex['E'] = new Complex(Math.E, 0);
+  Complex['INFINITY'] = new Complex(Infinity, Infinity);
```

Comment:
I would prefer calling this `ComplexInfinity` or simliar to make it clear that this is different from the real infinity that people are used to in JavaScript

Reply:
I suppose the same could be said for the `NAN`.  But since they are all CAPS and prefixed by the type (`Complex.INFINITY` and `Complex.NAN`) I am not sure the clarification is necessary.  Doesn't matter to me though...

Reply:
Even if `ComplexInfinity` is more unambiguous, I like `Complex.INFINITY` and `Complex.NAN` better, since you would normally use it exactly as "Complex.INFINITY" and so it states already what ComplexInfinity wants to express.

Reply:
I guess the question is how strongly do you feel about having the uppercase names. People are used to writing `NaN` rather than `NAN` so the upper case version may be confusing.